packages/agent-utils/src/index.ts

```diff
@@ -3,4 +3,4 @@ export * from "./sys";
 export * from "./scripts";
 export * from "./llm";
 export * from "./chunking";
-export * from "./memory";
\ No newline at end of file
+export * from "./vectors";
```

Comment:
Should we rename "vectors" to "embedding"? This seems to match the `EmbeddingApi` type. Similar to how we have the `llm/` directory where we define the `LlmApi`.